static/js/unread_ui.js

```diff
@@ -14,6 +14,22 @@ import * as unread from "./unread";
 import {notify_server_messages_read} from "./unread_ops";
 
 let last_mention_count = 0;
+let user_closed_mark_as_read_turned_off_banner = false;
+export function reset_mark_as_read_turned_off_banner() {
+    user_closed_mark_as_read_turned_off_banner = false;
+}
+
+export function hide_mark_as_read_turned_off_banner() {
+    // Use visibility instead of hide() to prevent messages on the screen from
+    // shifting vertically.
+    $("#mark_as_read_turned_off_banner").css({visibility: "hidden"});
+}
+
+export function notify_messages_remain_unread() {
+    if (!user_closed_mark_as_read_turned_off_banner) {
+        $("#mark_as_read_turned_off_banner").css({visibility: "visible"});
+    }
```

Comment:
This toggle can be done with toggling the `.invisible` CSS class from `app_components.css`, which would be better given our general policy of avoiding direct `.css()` toggles in JS code.